samples/drivers/mbox_data/README.rst

```diff
@@ -0,0 +1,88 @@
+.. zephyr:code-sample:: mbox_data
+   :name: MBOX Data
+   :relevant-api: mbox_interface
+
+   Perform inter-processor mailbox communication using the MBOX API with data.
+
+Overview
+********
+
+This sample demonstrates how to use the :ref:`MBOX API <mbox_api>`.
+
+Building and Running
+********************
+
+The sample can be built and executed on boards supporting MBOX.
+
+Building the application for mimxrt1160_evk_cm7
+*****************************************************
```

Comment:
looks like you're putting all headings at the same level?
Also, the "underline" should be the same length as the text

Reply:
Headings separated to correct sub-levels.
Underline changed to be same length as the text.